### DISCUSSION TITLE
fwmap bei RPI-RF-MOD nicht berücksichtigt

### DISCUSSION THREAD

**Discussion by jp112sdl** [OpenCCU/OpenCCU#1594] [🇩🇪 Fragen&Antworten] [answered]
Hi!

Mir ist heute früh aufgefallen, dass im `S48UpdateRFHardware`-Skript für das RPI-RF-MOD gar nicht geschaut wird, welche Firmware laut `fwmap` die zu verwendende ist. 

Stattdessen wird "erzwungenermaßen" die neueste Firmwaredatei im Verzeichnis `/firmware/RPI-RF-MOD` gesucht und verwendet.

Gibt es dafür einen bestimmten Grund? 
Ich wollte ein Firmware-Downgrade durchführen und das wie früher gewohnt über die `fwmap` steuern.
Hat mich ein wenig Zeit gekostet, bis ich dahintergekommen bin 😎

**Comment by jens-maus** [comment, marked as answer]
> Mir ist heute früh aufgefallen, dass im `S48UpdateRFHardware`-Skript für das RPI-RF-MOD gar nicht geschaut wird, welche Firmware laut `fwmap` die zu verwendende ist.

Die Nutzung der `fwmap` ist AFAIK im Grunde obsolete. Zumindest wenn man das neuere `hmip-copro-update.jar` tool von eq3 verwendet.

**Reply by jp112sdl** [reply]
Ok, ich merke mir einfach, dass bei RM nur 1 Datei im firmware-Verzeichnis zu liegen hat und die wird dann auch geflasht
